src/System.Collections.Sequences/System.Collections.Sequences.csproj

```diff
@@ -9,5 +9,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
+    <PackageReference Include="System.ValueTuple">
+      <Version>4.5.0-preview2-25707-02</Version>
```

Comment:
Can you use CoreFxStableVersion (i.e. 4.3.0)?

If not, consider adding a property SystemValueTypeVersion similar to SystemMemoryVersion in dependencies.props and update the version to latest.
https://github.com/dotnet/corefxlab/blob/master/tools/dependencies.props#L6

We will then have to update the update-dependencies program to update the version whenever a new package becomes available:
https://github.com/dotnet/corefxlab/blob/master/scripts/update-dependencies/Program.cs#L98

Reply:
will do